fixtures/generic/moving-bar-beam-10x40w.json

```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Moving Bar Beam 10x40W",
+  "shortName": "10X40W MOVING BEAM LAMP",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["Ebany Diaz"],
+    "createDate": "2023-06-06",
+    "lastModifyDate": "2023-06-06"
+  },
+  "comment": "GENERIC MODEL",
+  "links": {
+    "manual": [
+      "https://drive.google.com/file/d/18tp-LuZvn59sX0kjxZJixgaA9A4xcYdj/view?usp=drive_link"
```

Comment:
This link asks me to log in.

I don't know how "generic" this DMX mapping is, but interestingly, it does look pretty similar to the 7-channel mode of [the SHEHDS 10x40 RGBW bar](https://drive.google.com/file/d/1U3sNQPQtCuAvdBy6RtQ2pFyqMm_BORLp/view) (but not any LED bars already in OFL).